src/harlequin/config_wizard.py

```diff
@@ -175,6 +175,8 @@ def _prompt_for_path() -> tuple[Path, bool]:
         style=HARLEQUIN_QUESTIONARY_STYLE,
     ).unsafe_ask()
     path = Path(raw_path)
+    path = Path.expanduser(path)
```

Comment:
expanduser() is an instance method that takes no arguments. This should be
```suggestion
    path = path.expanduser()
```